vendor/github.com/containers/buildah/chroot/run.go

```diff
@@ -883,9 +883,9 @@ func setCapabilities(spec *specs.Spec, keepCaps ...string) error {
 	capMap := map[capability.CapType][]string{
```

Comment:
Why are vendor files changed directly? I think we need to properly update buildah and update go.mod to prevent unnecessary problems

Reply:
from what I could see from the other commits in [v1.6.4-rhel](https://github.com/containers/podman/tree/v1.6.4-rhel), vendor is modified directly without any vendoring